make.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+#
+
+PATH1=`pwd`
+echo $PATH1;
+find . -maxdepth 1 -type f -iname \*.sln > tmp1.out
+COUNT=`cat tmp1.out | wc -l`
+
+if [ $COUNT = 1 ]; then
+while read i; do
+PROJECT=$i;
+echo $PROJECT;
+done < tmp1.out
+        else
+            	if [ $COUNT = 0 ]; then
+                                echo "No .sln found"
+                        exit 1
+                fi
+        echo "ERROR: too many .sln files";
+        exit 1
+fi
```

Comment:
the code to find sln files is too convoluted and ugly hehe, let's just hardcode TcpEcho.sln, no big deal